### DISCUSSION TITLE
How often does the remote calendar update?

### DISCUSSION THREAD

**Discussion by HyperSigilMedia** [ivan-lednev/obsidian-day-planner#593] [Q&A]
Hi there! I really enjoy this plugin so far and I'm excited to see how it enriches my entire workflow to have such tight integration of my calendar into my notes. I'm running into an issue and I hope there's an easy solution. 
I see the appointments from my two Google calendars set up via ICS that had already been made the day that I imported the calendars, but I'm not seeing new appointments that have been made since. Is there a setting to change how often this remote file is updated? Is it updated automatically or is there some process we have to set up to do it? 
Thanks in advance for any assistance!

**Comment by ivan-lednev** [comment]
Hello, the calendars get updated automatically (right now it's every 5 minutes). You might check for errors in the console (`Control + Shift + I`)
